src/OpenTK/Platform/Android/AndroidDisplayDeviceDriver.cs

```diff
@@ -12,25 +12,37 @@ namespace OpenTK.Platform.Android
     internal class AndroidDisplayDeviceDriver : IDisplayDeviceDriver
```

Comment:
Well it kinda worked it would just always return a resolution of (x=0, y=0, width=0, height=0, refresh_rate=0, bit_depth=16). Which is also what the new code does.